build.sbt

```diff
@@ -2,10 +2,46 @@ import Dependencies._
 
 lazy val `akkaserverless-java-sdk` = project
   .in(file("."))
-  .aggregate(sdk, testkit, tck, codegenCore, codegenJava, codegenJavaCompilationTest)
+  .aggregate(sdkCore, sdkJava, sdkScala, testkitJava, tck, codegenCore, codegenJava, codegenJavaCompilationTest)
 
-lazy val sdk = project
-  .in(file("sdk"))
+//FIXME duplicating most settings in sdkCore, sdkJava, sdkScala for now, cleanup later.
+lazy val sdkCore = project
+  .in(file("sdk/core"))
+  .enablePlugins(AkkaGrpcPlugin, BuildInfoPlugin, PublishSonatype)
+  .settings(
+    name := "akkaserverless-java-sdk-core",
+    crossPaths := false,
+    buildInfoKeys := Seq[BuildInfoKey](
```

Comment:
do we need `BuildInfo` in all 3 artifacts?

Reply:
I'm not sure, first step was to copy paste, and remove later. I can remove them from core. I guess they should be there for scala and java sdk.